### ISSUE TITLE
Refactor loading messages

### ISSUE THREAD

**Issue by zebdelk** [cedardevs/onestop#549]
Currently the redux store has just one loading indicator, so screen reader users cannot easily be informed of what exactly has loaded. The message read dynamically is just "load complete" right now - once we change our store to correctly reflect which requests are pending and complete, we can report "N collections loaded", "M granules loaded", "collection detail loaded", and other appropriate messages.

**Comment by zebdelk** [comment]
This was fixed in a refactor related to adding granule filtering.
